### PR TITLE
Add changelog of v2.25.0 

### PR DESCRIPTION
**Pull request by chhluo**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add changelog of v2.25.0 .

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +171,7 @@ Results and models are available in the [model zoo](docs/en/model_zoo.md).
             <li><a href="configs/yolox">YOLOX (CVPR'2021)</a></li>
             <li><a href="configs/deformable_detr">Deformable DETR (ICLR'2021)</a></li>
             <li><a href="configs/tood">TOOD (ICCV'2021)</a></li>
+            <li><a href="configs/ddod">DDOD (MM'2021)</a></li>
```
Comment (shinya7y) [inline README.md:174]: `MM` could cause confusion with other `MultiMedia` such as `IEEE MultiMedia`.
`ACMMM` or `ACM MM` would be more appropriate abbreviations.
https://openreview.net/group?id=acmmm.org/ACMMM/2022/Conference
https://twitter.com/acmmm2022

File: docs/en/compatibility.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
 # Compatibility of MMDetection 2.x
 
+## MMDetection 2.25.0
+
+Inorder to support Mask2Former for instance segmentation, the original config files of Mask2Former for panpotic segmentation need to be renamed [PR #7571](https://github.com/open-mmlab/mmdetection/pull/7571).
```
Comment (shinya7y) [inline docs/en/compatibility.md:5]: `In order`

File: README.md [hunk 3]
```diff
@@ -74,11 +74,38 @@ This project is released under the [Apache 2.0 license](LICENSE).
 
 ## Changelog
 
-**2.24.1** was released in 30/4/2022:
+**2.25.0** was released in ???/5/2022:
 
-- Support [Simple Copy Paste](configs/simple_copy_paste)
-- Support automatically scaling LR according to GPU number and samples per GPU
-- Support Class Aware Sampler that improves performance on OpenImages Dataset
+- Support [ConvNeXt](configs/convnext)
```
Comment (ZwwWayne) [inline README.md:79]: squeeze them into three lines.
Support ConvNext, DDORD, SOLOv2
Support Mask.....
Rename...

File: docs/en/changelog.md [hunk 4]
```diff
@@ -1,5 +1,121 @@
...
+  - Rewrite the [installation guide](docs/en/get_started.md) (#7897)
+  - [Useful hooks](docs/en/tutorials/useful_hooks.md) (#7810)
+  - Fix heading anchor in documentation  (#8006)
+  - Replace `markdownlint` with `mdformat` for avoiding installing ruby (#8009)
+
+#### Contributors
+
+A total of 18 developers contributed to this release.
+
+Thanks @ZwwWayne, @Wsnhg, @Solomiya, @Luting Wang, @ChenXF, @Czm369, @Chenastron, @Cedric Luo, @Shanyaliux, @Kai Chen, @Y-M-Y, @jbwang1997, @Haian Huang(深度眸), @Irving.Gao, @zhanggefan, @BigDong, @Peter Vennerstrom, @Ayush Thakur
```
Comment (ZwwWayne) [inline docs/en/changelog.md:117]: should all use github ID, some of them uses names, which is incorrect.

File: docs/en/changelog.md [hunk 5]
```diff
@@ -1,5 +1,122 @@
 ## Changelog
 
+### v2.25.0 (31/5/2022)
+
+#### Highlights
+
+- Support [ConvNeXt](https://arxiv.org/abs/2201.03545) (#7281)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:7]: highlights do not need to post PR ID, and they should be consistent with those in README.md.

File: docs/en/changelog.md [hunk 6]
```diff
@@ -1,5 +1,122 @@
...
+- Support [DDOD](https://arxiv.org/abs/2107.02963) (#7279)
+- Support [SOLOv2](https://arxiv.org/abs/2003.10152) (#7441)
+- Support [Mask2Former](https://arxiv.org/abs/2112.01527) for instance segmentation (#7571, #8032)
```
Comment (hhaAndroid) [inline docs/en/changelog.md:10]: miss wandbloggerhook

File: docs/en/changelog.md [hunk 7]
```diff
@@ -1,5 +1,122 @@
...
+
+#### Bug Fixes
+
+- Enable YOLOX training on different devices (#7912)
+- Fix path issue of `docs/en/stat.py` (#6696)
+- Fix the log plot error when evaluation with `interval != 1` (#7784)
+- Fix windows CI (#8046)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:49]: unnecessary

File: docs/en/changelog.md [hunk 8]
```diff
@@ -1,5 +1,122 @@
...
+
+- Support [ConvNeXt](https://arxiv.org/abs/2201.03545) (#7281)
+- Support [DDOD](https://arxiv.org/abs/2107.02963) (#7279)
+- Support [SOLOv2](https://arxiv.org/abs/2003.10152) (#7441)
+- Support [Mask2Former](https://arxiv.org/abs/2112.01527) for instance segmentation (#7571, #8032)
+
+#### Bug Fixes
+
+- Enable YOLOX training on different devices (#7912)
+- Fix path issue of `docs/en/stat.py` (#6696)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:47]: unnecessary

File: docs/en/changelog.md [hunk 9]
```diff
@@ -1,5 +1,122 @@
...
+
+#### Improvements
+
+- Support reading `gpu_collect` from `cfg.evaluation.gpu_collect` (#7672)
+
+- Speedup the Video Inference by Accelerating data-loading Stage (#7832)
+
+- Support replacing the `${key}` with the value of `cfg.key` (#7492)
+
+- Accelerate analyzing result. The evaluation time on results (coco val2017) generated by `retinanet_r50_fpn_1x_coco` takes 10~15 mins. (#7891)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:60]: Accelerate analyzing result -> Accelerate result analysis in xxx.py. The evaluation time is speedup by xx times and only tasks xx minutes now.

File: docs/en/changelog.md [hunk 10]
```diff
@@ -1,5 +1,122 @@
...
+
+- Add parameters `block_dilations` in DilatedEncoder (#7812)
+
+- Support panoptic segmentation result analysis (#7922)
+
+- Add `AvoidOOM` to avoid OOM (#7434, #8091)
```
Comment (ZwwWayne) [inline docs/en/changelog.md:66]: Put this to the top.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**